runtime/compiler/control/JitDump.cpp

```diff
@@ -483,9 +483,98 @@ runJitdump(char *label, J9RASdumpContext *context, J9RASdumpAgent *agent)
 
    try
       {
-      // if our compinfo is null, we are an application thread
+      // Process user defined options first. The user can specify any method signature to be recompiled at an arbitrary
+      // dump event. This is typically used for debugging purposes so we process this option first if it exists.
+      if (agent->dumpOptions != NULL)
+         {
+         // TODO: This is a total hack. Surely there must be an API out there where we can supply a full method
+         // signature and get back a J9Method* if it exists? Right? I've searched but I cannot find this API. I
+         // will tag this in a review and hopefully someone knows the right answer here.
```

Comment:
While I empathize with your frustration, perhaps this is more appropriate as a comment in the PR at this source line for other reviewers rather than committing it into the code itself (where it will live on indefinitely).  An issue tracking finding and implementing the proper solution to improve this interim implementation is probably warranted too.

Reply:
Sorry about that, this comment will never be merged into the branch. It was more a reminder for myself to open up a discussion. Similar TODO was left for another discussion, which once resolved will be removed.

Converting this to WIP draft PR to prevent accidental merges.

Reply:
This should be fixed now in 1e3a722.